<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdg_resource_browser</class>
 <widget class="QWidget" name="wdg_resource_browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyQGIS Icons Browser</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter:</string>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsFilterLineEdit" name="tbFilter">
       <property name="nullValue">
        <string/>
       </property>
       <property name="showSearchIcon">
        <bool>true</bool>
       </property>
       <property name="qgisRelation" stdset="0">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnUseRegex">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCaseSensitive">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnReload">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_settings">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QSplitter::handle {
    image: url(images/splitter.png);
}

QSplitter::handle:vertical {
    height: 5px;
}

QSplitter::handle:pressed {
    url(images/splitter_pressed.png);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ResourceTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageImage">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ResourceGraphicsView" name="graphicsView">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(240, 240, 240);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageText">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="info">
     <property name="styleSheet">
      <string notr="true">text-color:rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="optionUseRegex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true">.*</string>
   </property>
   <property name="toolTip">
    <string>Activate to use regular expressions</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset>
     <normaloff>:/images/themes/default/mActionReload.svg</normaloff>:/images/themes/default/mActionReload.svg</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload</string>
   </property>
  </action>
  <action name="optionCaseSensitive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true">cC</string>
   </property>
   <property name="toolTip">
    <string notr="true">Activate to filter case-sensitive</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ResourceTableView</class>
   <extends>QTableView</extends>
   <header>pyqgis_resource_browser.core.resource_table_view</header>
  </customwidget>
  <customwidget>
   <class>ResourceGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqgis_resource_browser.gui.resource_browser</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qpsresources.qrc"/>
 </resources>
 <connections/>
</ui>
